lib/player.rb

```diff
@@ -0,0 +1,69 @@
+require_relative 'score'
+
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :tiles
+    def initialize(name)
+      @name = name
+      @plays = []
+      @tiles = []
+    end
+
+
+    def play(word)
+      if won?
+        return false
+      else
+        @plays << word
+        return Scrabble::Score.score(word)
+      end
+    end
+    # def play(word)
+    #   valid_play = true
+    #   word = word.upcase
+    #   word.split("").each do |i|
+    #     if !@tiles.include? i
+    #       valid_play = false
+    #     end
+    #   end
+    #   if valid_play
+    #     if won?
+    #       return false
+    #     else
+    #       @plays << word
+    #       return Scrabble::Score.score(word)
+    #     end
+    #   else
+    #     puts "You must use the tiles you have"
+    #   end
+    # end
+
+
+    def total_score
+      total_so_far = 0
+      @plays.each do |i|
+        total_so_far += Scrabble::Score.score(i)
+      end
+      return total_so_far
+    end
+
+
+    def won?
+      total_score > 100 ? true : false
+    end
```

Comment:
You don't need the ternary operator here - because `total_score > 100` already evaluates to a boolean, you can just return that.